snmp/assets/dashboards/interface_performance.json

```
@@ -0,0 +1,521 @@
+        "tick_pos": "50%",
+        "tick_edge": "left"
+      }
+    },
+    {
+      "id": 11,
+      "layout": { "x": 135, "y": 7, "width": 31, "height": 34 },
+      "definition": {
+        "type": "note",
+        "content": "Bandwidth is calculation measured as the rate at which traffic is sent or received on a specific port over the max speed that traffic can be sent. \n\nWe use the following formula: \n\ninbound % bandwidth\n![inbound-eq](https://microsoft.codecogs.com/svg.latex?=\\frac{snmp.ifHCInOctets.rate%20*8*100}{snmp.ifHighSpeed%20*%201000000})\n\noutbound % bandwidth\n![inbound-eq](https://microsoft.codecogs.com/svg.latex?=\\frac{snmp.ifHCOutOctets.rate%20*8*100}{snmp.ifHighSpeed%20*%201000000})\n\nDatadog Agent version 7.25+ includes a packaged metric that calculates bandwidth:\n\n* `snmp.ifBandwidthInUsage.rate`\n* `snmp.ifBandwidthOutUsage.rate`",
+        "background_color": "blue",
+        "font_size": "14",
+        "text_align": "left",
+        "show_tick": true,
+        "tick_pos": "50%",
+        "tick_edge": "left"
+      }
+    },
+    {
+      "id": 12,
+      "layout": { "x": 21, "y": 143, "width": 56, "height": 32 },
+      "definition": {
+        "title": "Interface Status",
+        "title_size": "16",
+        "title_align": "left",
+        "type": "query_table",
+        "requests": [
+          {
+            "aggregator": "avg",
+            "cell_display_mode": ["bar"],
+            "conditional_formats": [
+              {
+                "comparator": "<",
+                "palette": "black_on_light_green",
+                "value": 2
+              },
+              {
+                "comparator": "<=",
+                "palette": "black_on_light_red",
+                "value": 2
+              },
+              {
+                "comparator": "<=",
+                "palette": "black_on_light_yellow",
+                "value": 3
+              },
+              {
+                "custom_bg_color": "#cfcdcd",
+                "comparator": ">",
+                "palette": "custom_bg",
+                "value": 3
+              }
+            ],
+            "q": "avg:snmp.ifOperStatus{$snmp_host,$interface,$snmp_device} by {snmp_device,snmp_host,interface}",
+            "alias": "oper status",
+            "limit": 50,
+            "order": "desc"
+          }
+        ],
+        "has_search_bar": "auto"
+      }
+    },
+    {
+      "id": 13,
+      "layout": { "x": 21, "y": 130, "width": 113, "height": 6 },
+      "definition": {
+        "type": "note",
+        "content": "State",
+        "background_color": "orange",
+        "font_size": "18",
+        "text_align": "center",
+        "show_tick": false,
+        "tick_pos": "50%",
+        "tick_edge": "left"
+      }
+    },
+    {
+      "id": 14,
+      "layout": { "x": 21, "y": 137, "width": 56, "height": 5 },
+      "definition": {
+        "type": "note",
+        "content": "Current State",
+        "background_color": "orange",
```

```suggestion
        "background_color": "blue",
```
Sub category headers are blue. https://datadoghq.dev/integrations-core/guidelines/dashboards/#subcategories